presto-main/src/main/java/io/prestosql/sql/ExpressionUtils.java

```diff
@@ -225,7 +233,35 @@ public static Expression combineDisjunctsWithDefault(Metadata metadata, Collecti
             return TRUE_LITERAL;
         }
 
-        return disjuncts.isEmpty() ? emptyDefault : or(disjuncts);
+        ImmutableMultimap.Builder<Expression, Expression> builder = ImmutableMultimap.builder();
```

Comment:
if we keep it here, I think `io.prestosql.sql.ExpressionUtils#extractDisjuncts` should be symmetric.